client/src/canvas/constants.ts

```diff
@@ -3,3 +3,4 @@ export const CANVAS_WIDTH = 1000;
 export const CANVAS_HEIGHT = 1000;
 export const DEFAULT_ZOOM = 41.0;
 export const ZOOM_CHANGE_FACTOR = 10;
+export const TIME_BETWEEN_UPDATES_MS = 300000;
```

Comment:
Are we gonna hardcode this on the client or get it from the server?

Reply:
I think we'll probably leave it as a constant on the front end